make/utils_exposed.cmake

```diff
@@ -145,12 +145,45 @@ function (_DR_check_if_linker_is_gnu_gold var_out)
   set(${var_out} ${is_gold} PARENT_SCOPE)
 endfunction (_DR_check_if_linker_is_gnu_gold)
 
+# Takes in a target and returns the expected target name.
```

Comment:
You mean the target *path*, not the name ("target name" is just that: the name within cmake, like "dynamorio", with no necessary relation to a file).

Reply:
Takes in a target and returns the expected full target path incl. output name.